mmtrack/datasets/mot_challenge_dataset.py

```diff
@@ -6,6 +6,7 @@
 import mmcv
 import motmetrics as mm
 import numpy as np
+import trackeval
```

Comment:
follow the [example](https://github.com/open-mmlab/mmtracking/blob/master/mmtrack/core/evaluation/eval_sot_vot.py#L10) to import trackeval